### DISCUSSION TITLE
There has been no update for a long time. What happened?

### DISCUSSION THREAD

**Discussion by lisyclover** [microsoft/WSA#532] [General]
There has been no update for a long time. What happened?

**Comment by MustardChef** [comment]
Hopefully it's due to a huge update that they are working on (🤞), or we must fear for the worse.

Update: RIP WSA!!!!!!!!!

**Reply by lisyclover** [reply]
I hope the project goes well. I have been using WSA since it was released, so that my surface pro x has the characteristics of an Android tablet.

**Comment by RebelliousX** [comment]
I don't think MS cares about WSA anymore.

**Reply by ghost** [reply]
I think so

**Comment by Morrigan-Ship** [comment]
*WSA will expire
*WSA will Full Remake because its bad design in backend
*its because of Android 14 and 15 update
**they are working on boot option for WSA like bootloader custom recovery like twrp flash custom image WSA**

may be one of them i love last option :))

**Comment by RebelliousX** [comment]
And WSA is officially dead.

**Comment by Morrigan-Ship** [comment]
ops!

**Comment by agussaputrasijabat** [comment]
Naniii
